test/snippet/range/views/test_back_inserter_zip.cpp

```diff
@@ -0,0 +1,20 @@
+#include <vector>
+
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/std/concepts>
+#include <seqan3/std/ranges>
+
+#include <range/v3/view/zip_with.hpp>
+
+
+int main()
+{
```

Comment:
what does this snippet have to do with all of this?

Reply:
boah. Why is this added? :rage3: